### ISSUE TITLE
Add class for combined concepts

### ISSUE THREAD

**Issue by nichtich** [linked-statistics/xkos#132]
Concept Schemes used for subject index can be based on pre-coordinated concepts (e.g. a list of countries combined with other concepts to form country-specific concepts) or they can be used for post-coordinated indexing (a resource is indexed with multiple concepts). SKOS has no method to express such combined (aka compound) concepts. In JSKOS we introduced [concept bundles](https://gbv.github.io/jskos/jskos.html#concept-bundles) but there is no RDF class yet. I propose to introduce a new class "CombinedConcept" to XKOS. This class could be a superclass of [xkos:ConceptAssociation](http://www.ddialliance.org/Specification/XKOS/1.0/OWL/xkos.html#correspondences).

* **URI:** xkos:CombinedConcept
* **Description**: A set of concepts that together form a new concept.

**Comment by duffgui** [comment]
The proposal seems to conflate two rather different things:

- a concept that is composed of other concepts, and
- a relationship/correspondence between concepts.
Those deserve different modeling.

The definition of `xkos:CombinedConcept` is quite generic. Here again, the main purpose of XKOS is essentially SKOS with additional semantics for statistical classifications. Nothing in that definition is inherently statistical or classification-specific.

Furthermore, it seems to me that there is also a problem with making it a superclass of `xkos:ConceptAssociation`. Indeed, a combined concept and a concept association/correspondence have different directions of meaning.
For example:
```
France + Male
       ↓
Males in France
```
is a sort of composition relationship.
Whereas a correspondence is more like:
`Concept A  ←→  Concept B`
possibly with a mapping relationship between concepts in two schemes. The fact that a correspondence can involve multiple concepts does not necessarily make the correspondence itself a combined concept.

Following on the "Male in France" example, one should keep in mind that in the statistical context, the geography is represented by dedicated objects in the [GSIM](https://unece.github.io/GSIM-2.0/#universe) and [DDI](https://docs.ddialliance.org/DDI-Lifecycle/3.3/model/item-types/Universe/?highlight=universe) models (the Population and Universe) which are central and widely-used in the official statistics area.

However, `skos:Collection` may actually be the closest existing mechanism.
If the "bundle" isn't intended to be a new conceptual entity in its own right, but merely a way of saying: 

> "these concepts are to be treated together for indexing"

then `skos:Collection` is probably enough.

For example:

```
ex:FranceMaleBundle
    a skos:Collection ;
    skos:member ex:France ;
    skos:member ex:Male 
```

That gives you a completely standard SKOS representation, and it avoids creating another class. Nevertheless, the crucial question therefore becomes: Is the bundle itself a concept?
- If no, I'd strongly prefer skos:Collection.
- If yes, then I think SKOS currently has a genuine modeling gap which is much broader than XKOS.

Finally, I guess there is a subtle distinction between a combined concept (e.g. France + Male) and a representation of a combination of concepts (Males in France).
The latter could be considered as a `skos:Concept.` What is special about it isn't its class, but its definition as a composition of other concepts.

That suggests the following modelling:
```
ex:MalesInFrance
    a skos:Concept ;
    ex:composedOf ex:France, ex:Male .
```
rather than:
```
ex:MalesInFrance
    a xkos:CombinedConcept ;
    ...
```
In other words, the compositional relationship may be the thing that needs vocabulary, rather than a new class. Here again, that vocabulary needs not necessarily belong to XKOS, but to some other more generic such as SKOS.

If you are O.K. with those remarks, I would be inclined to close this issue.
